README.md

```diff
@@ -150,13 +150,15 @@ Lemmy is free, open-source software, meaning no advertising, monetizing, or vent
 
 ## Contributing
 
-- [Contributing instructions](https://join-lemmy.org/docs/en/contributing/contributing.html)
+<!-- - [Contributing instructions](https://join-lemmy.org/docs/en/contributing/contributing.html)
 - [Docker Development](https://join-lemmy.org/docs/en/contributing/docker_development.html)
-- [Local Development](https://join-lemmy.org/docs/en/contributing/local_development.html)
+- [Local Development](https://join-lemmy.org/docs/en/contributing/local_development.html) -->
+
+- In order to contribute to Lemmy, you will need to fork this repository. You can then make changes to the code and submit a pull request. Changes will be reviewed and merged into the main repository if accepted.
```

Comment:
Fix the links instead of commenting them out. Text should go directly in the docs.